flytekit/models/core/resource.py

```diff
@@ -0,0 +1,289 @@
+import typing
+
+from flyteidl.core import resource_pb2 as _resource_pb2
+
+from flytekit.models import common as _common
+
+
+class WorkerGroupSpec(_common.FlyteIdlEntity):
+    def __init__(
+        self,
+        group_name: str,
+        replicas: int,
+        min_replicas: typing.Optional[int] = 0,
+        max_replicas: typing.Optional[int] = None,
+        compute_template: typing.Optional[str] = None,
+        image: typing.Optional[str] = "rayproject/ray:1.8.0",
```

Comment:
can we not use the `container_image` directly? or this for the worker image. 
Even if so, can we not re-use the worker image?

Reply:
Updated it, Now we use [task image](https://github.com/flyteorg/flyteidl/blob/7b1fd01ff429d28755765204f00dea57598d8851/protos/flyteidl/core/tasks.proto#L154) by default for the head node and worker nodes, so we have to use `rayproject/ray:1.8.0` as our base image